contracts/test/integration/IntegrationTestPriceBoundPSMUSDC.t.sol

```diff
@@ -143,10 +144,11 @@ contract IntegrationTestPriceBoundPSMUSDCTest is DSTest {
     }
 
     /// @notice PSM is set up correctly and view functions are working
-    function testGetMintAmountOut() public {
-        uint256 amountUSDCIn = 100e18;
-        uint256 currentPegPrice = oracle.getCurrentOraclePrice();
+    function testGetMintAmountOut(uint256 amountUSDCIn) public {
+        if (amountUSDCIn > usdc.balanceOf(address(this))) return;
+        amountUSDCIn = amountUSDCIn * 1e18;
 
+        uint256 currentPegPrice = oracle.getCurrentOraclePrice();
```

Comment:
need to test getMintAmountOut against both the new and old PSM in the same function to ensure their output is the same.